working/templates/gqi-ad-hoc-data-source-solution/Dlls/ABOUT.md

```diff
@@ -0,0 +1 @@
+This folder contains DLL files that should be placed under the ProtocolScripts folder of DataMiner for this solution to work correctly.
```

Comment:
Not sure that this works actually. I think I remember people needing to add assemblies to Files to get dll's to work because it didn't load assemblies from protocolscripts. To be checked with Seba/Wim

Reply:
Indeed, currently this doesn't function as expected. Removed the folder.